src/utils/options.js

```diff
@@ -3,8 +3,9 @@ function getOption({ opts }, name, defaultValue = true) {
 }
 
 export const useDisplayName = (state) => getOption(state, 'displayName')
-export const useSSR = (state) =>  getOption(state, 'ssr', false)
-export const useFileName = (state) =>getOption(state, 'fileName')
+export const useSSR = (state) => getOption(state, 'ssr', false)
+export const useFileName = (state) => getOption(state, 'fileName')
 export const useMinify = (state) => getOption(state, 'minify')
 export const useCSSPreprocessor = (state) => getOption(state, 'preprocess', false) // EXPERIMENTAL
 export const useTranspileTemplateLiterals = (state) => getOption(state, 'transpileTemplateLiterals')
+export const useIncreaseSpecificityClass = state => getOption(state, 'specificityClass', null);
```

Comment:
Usually these are called "namespace classes" (not to be confused with CSS namespaces which serve a different purpose). Perhaps that terminology would be more clear.